parsers/forth.c

```diff
@@ -0,0 +1,76 @@
+/*
```

Comment:
We don't need this file anymore.

Reply:
yes, I understand. But couldn't it be possible to skip it anyway? Geany told me they wanted it to be pushed upstream before integrating it into their code, and on the other hand they can't handle the .ctags / optlib tags...

Reply:
See https://github.com/geany/geany/issues/3557 . 

The .ctags is translated to C. So technically, it is possible to integrate the .ctags based Forth parser into Geany.

Let's see how Geany people say.

Reply:
@farvardin we got a positive answer from Geany developers introducing an optlib-based parser to Geany. 

Could you remove parsers/forth.c ?

If we recognize the optlib-based implementation is not enough, you can open a pull request implementing the parser with crafted-C-code in the future. We can re-implement the parser safely because we have the test case.

Reply:
@masatake ok, I've done it. There was a conflict merge, I hope I haven't deleted some of your code, but it seems fine from my side